apps/src/code-studio/teacher.js

```diff
@@ -176,10 +176,12 @@ function renderViewAsToggle(element) {
 function renderTeacherPanelSections(element) {
   ReactDOM.render(
     <Provider store={getStore()}>
-      <SectionSelector
-        reloadOnChange={true}
-        alwaysShow={true}
-      />
+      <span style={{margin: 10}}>
```

Comment:
There are several places with spans/styles around SectionSelector. Should the component have a style prop instead?

Reply:
Good question. I think you might be right that doing so could be cleaner. I will explore that.